testdata/dnn/onnx/generate_quantized_onnx_models.py

```diff
@@ -33,8 +33,8 @@ def quantize_and_save_model(name, input, model, act_type="uint8", wt_type="uint8
                     activation_type=type_dict[act_type], weight_type=type_dict[wt_type])
 
     os.remove(float_model_path)
-    os.remove(os.path.join("models", "dummy-opt.onnx"))
-    os.remove("augmented_model.onnx")
+    # os.remove(os.path.join("models", "dummy-opt.onnx"))
+    # os.remove("augmented_model.onnx")
```

Comment:
Most probably it should be reverted.

Reply:
These files did not exist with the latest onnxruntime. Most probably they are handled well since a version update.

Reply:
These files did not exist with the latest onnxruntime. Most probably they are handled well since a version update.